### DISCUSSION TITLE
save flags in a configuration file

### DISCUSSION THREAD

**Discussion by teto** [sshuttle/sshuttle#882] [Q&A] [answered]
silly question but my typical sshuttle invocation involves many flags and I provide the command for other persons too.

I looked through strace and --help but could not find a way to load all flags from a file. Alternatively I could wrap the sshuttle binary in a script that passes default flags.

**Comment by brianmay** [comment, marked as answer]
Does https://sshuttle.readthedocs.io/en/stable/manpage.html#configuration-file help?

**Reply by teto** [reply]
arf I dont know how I missed that I went to readthedocs I promise. (would be nice to include it in --help too).  The worst is I think I blindly tried it since several software use that approach but got confused with an unrelated error and discarded the approach. Thanks !
